libs/dokobit-signing/src/lib/signing.service.spec.ts

```diff
@@ -0,0 +1,133 @@
+import { mock } from 'jest-mock-extended'
+import { Base64 } from 'js-base64'
+import { createHash } from 'crypto'
+
+import { Test, TestingModule } from '@nestjs/testing'
+
+import { Logger, LOGGER_PROVIDER } from '@island.is/logging'
+
+import { SigningService } from './signing.service'
+
+const testOptions = {
+  url: 'Test Url',
+  accessToken: 'Test Access Token',
+}
+const testMobileNumber = '1111111'
+const testMessage = 'Test Message'
+const testContact = 'Test Contact'
+const testLocation = 'Test Location'
+const testDocumentName = 'Test Document Name'
+const testDocumentContent = 'Test Document Content'
+
+const testSignPath = `sign.json?access_token=${testOptions.accessToken}`
+const testSignResponse = {
+  status: 'ok',
+  control_code: 'Test Control Code', // eslint-disable-line @typescript-eslint/camelcase
+  token: 'Test Document Token',
+}
+const testStatusPath = `sign/status/${testSignResponse.token}.json?access_token=${testOptions.accessToken}`
+const testSignedDocumentContent = 'Test Content'
+const testStatusResponse = {
+  status: 'ok',
+  file: {
+    name: testDocumentName,
+    digest: createHash('sha256')
+      .update(testSignedDocumentContent)
+      .digest('hex'),
+    content: testSignedDocumentContent,
+  },
+}
+const postMock = jest.fn(function(
+  path: string,
+  body?: Body, // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
Why include these parameters if we don't use them?

Reply:
This is a mock of a function and the mock does not grab the call if I exclude the arguments. Later, I match calls to the mocked function.